packages/backend-defaults/src/types.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { AnyServiceFactory } from '@backstage/backend-plugin-api';
+
+/**
+ * @public
+ */
+export interface CreateBackendOptions {
```

Comment:
I feel this could just as well have lived in the same file as createBackend itself - it's so strongly tied to it and has no reusability